csds-client/client/util.go

```diff
@@ -0,0 +1,201 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
Same here, named imports could go in a separate block for readability

Reply:
Solved.